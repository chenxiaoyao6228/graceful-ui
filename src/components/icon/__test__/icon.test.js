import { render } from "@testing-library/vue";
import Icon from "../icon.vue";

describe("Icon", () => {
  it("icon type", () => {
    const { container } = render(Icon, {
      propsData: {
        type: "download",
      },
    });
    expect(container.innerHTML).toBe(
      '<svg class="g-icon"><use xlink:href="#icon-download" aria-hidden="true"></use></svg>'
    );
  });
});
