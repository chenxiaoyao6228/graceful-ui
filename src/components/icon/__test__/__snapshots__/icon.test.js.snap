// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icon icon type 1`] = `
<svg class="g-icon">
  <use xlink:href="#icon-download" aria-hidden="true"></use>
</svg>
`;
